tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -90,11 +96,16 @@ func TestWSU(t *testing.T) {
 	require.NotEmptyf(t, playbookPath, "WSU_PATH environment variable not set")
 	require.NotEmptyf(t, clusterAddress, "CLUSTER_ADDR environment variable not set")
 
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	require.NoError(t, err, "Could not build kubeconfig")
+	k8sclientset, err = kubernetes.NewForConfig(config)
```

Comment:
Please use the `getKubeClient()` helper.

Reply:
This helper isnt within this fille, but is a private function within the `internal/test` package. I had talked with @ravisantoshgudimetla about future work involving reusing the test framework and moving the wsu tests to the `internal/test` package, which was planned to do in a future ticket. 

For now I can create a new `getKubeClient` function here, to make the code more readable if thats desired

Reply:
That would make things more clear. But please do the require in the call site.